webpack/js/components.js

```diff
@@ -71,7 +71,7 @@ export const IssueCard = {
   computed: {
     dateCreated() {
       const dateCreated = new Date(this.issue.createdAt * 1000)
-      return dateCreated.toISOString()
+      return dateCreated.toISOString().substring(0,10)
```

Comment:
Instead of hardcoding `10`, can we not do `.indexOf('T')`? That seem's cleaner imo.

Reply:
Ok, can I use split instead of substring as it will be much better?

Reply:
Whichever you prefer.